### PR TITLE
Added theme options and help flag.

### PR DESCRIPTION
**Pull request by KisuYami**
Added:
	- Help flag, -h
	- Theme flag, -f
	- Updated readme file

### CODE REVIEW

**Comment by rafaelmardojai** [comment]
I'm not sure about the approach to change colors. This will cause trouble when someone tries to update the theme with git.

**Comment by KisuYami** [comment]
I think too, a better approuach would be to set this config on `userChrome.css`, but i don't know how to do it.

Something like this in the userChrome file:
```css
ACCENT_COLOR=<COLOR>
```
And then use it in the `dark.css` and `light.css` instead of the hardcoded hex value.

**Comment by KisuYami** [comment]
I changed the way that the theme is set, now it won't cause problems with git updates.

File: scripts/install.sh [hunk 1]
```diff
@@ -51,6 +61,16 @@ cp -R $THEMEDIRECTORY $PWD
...
 sed -i '1s/^/@import "firefox-gnome-theme\/userChrome.css";\n/' userChrome.css
 
+if [ $THEME != "DEFAULT" ]; then
+	if [ $THEME = "yaru" ]; then
+		echo "Setting $THEME theme."
+		echo '@import "firefox-gnome-theme\/theme/colors/dark-yaru.css";' >> userChrome.css
+		echo '@import "firefox-gnome-theme\/theme/colors/light-yaru.css";' >> userChrome.css
+	fi
+else
+	echo "No theme set, using default adwaite."
```
Comment (rafaelmardojai) [inline scripts/install.sh:71]: There's a typo in `adwaita`

File: scripts/install.sh [hunk 2]
```diff
@@ -60,4 +80,4 @@ if ! ln -s chrome/firefox-gnome-theme/configuration/user.js user.js ; then
 	echo "Please, manually copy theme's user.js contents to yours."
 fi
 
-echo "Done."
+echo "Done."
```
Comment (rafaelmardojai) [inline scripts/install.sh:83]: Preserve the new line at the end.

**Comment by KisuYami** [comment]
I should have tested it sooner, but this new approach breaks the dark theme.

**Comment by rafaelmardojai** [comment]
> I should have tested it sooner, but this new approach breaks the dark theme.

That's strange. Do you mean it stay with Adwaita colors or is entirely broken?

**Comment by KisuYami** [comment]
The dark theme won't work, but i already fixed it. If the white theme is imported after the dark theme, the dark theme wont work.

**Comment by rafaelmardojai** [comment]
Yeah, I see.

**Pull request merged**